settings.gradle

```diff
@@ -1,5 +1,6 @@
 include ":api"
 include ":app"
+include ":app:test_plugins"
```

Comment:
`test_plugins` and `integ-test` are following different naming conventions. I'd pick one and use it.

Reply:
I'd prefer the underscore format. Not sure where other people lie on this

Reply:
Idk. I'd just make it consistent.